src/geneweaver/api/core/config_class.py

```diff
@@ -38,7 +42,7 @@ def assemble_db_connection(
         "openid profile email": "read",
     }
     JWT_PERMISSION_PREFIX: str = "approle"
-    AUTH_CLIENT_ID: str = "oVm9omUtLBpVyL7YfJA8gp3hHaHwyVt8"
+    AUTH_CLIENT_ID: str = "T7bj6wlmtVcAN2O6kzDRwPVFyIj4UQNs"
```

Comment:
I assume this is not a hard coded cred. There is no way that much yaml and then still have things like this if so.

Reply:
It's the dev client ID. It's not a secret and makes sense to put in ad a default. 

The specific line change here relates to what made sense to put into the kubernetes config maps.